frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -0,0 +1,134 @@
+    Given user is at Topology page
+    And user has a new workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Add Health Checks
+    Then user will be redirected Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Add Readiness Probe Health Check to Deployments from Sidebar: MH-05-TC02
+    Given user is at Topology page
+    And user has a new workload of Deployment type resource with no Health Checks added
+    When user clicks on the workload to open the sidebar
+    And user clicks on Add Health Checks
+    And user clicks on Add Readiness Probe
+    And user clicks on right button
+    And user clicks on Add button
+    Then user will be redirected to Topology page
+    And user will see Readiness Probe added on the Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Add Liveness Probe Health Check to Deployment Configs from Actions dropdown Sidebar: MH-05-TC02
+    Given user is at Topology page
+    And user has a new workload of Deployment Config type resource with no Health Checks added
+    When user clicks on the workload to open the sidebar
+    And user clicks on Actions dropdown
+    And user clicks on Add Health Checks
+    And user clicks on Add Liveness Probe
+    And user clicks on right button
+    And user clicks on Add button
+    Then user will be redirected to Topology page
+    And user will see Liveness Probe added on the Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Add Startup Probe Health Check to Deployments from Context Menu: MH-05-TC02
+    Given user is at Topology page
+    And user has a new workload of Deployment type resource with no Health Checks added
+    When user right clicks on the workload to open the Context Menu 
+    And user clicks on Add Health Checks
+    And user clicks on Add Startup Probe
+    And user clicks on right button
+    And user clicks on Add button
+    Then user will be redirected to Topology page
+    And user will see Startup Probe added on the Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Edit Health Checks option for Knative Service through Context Menu: MH-05-TC02
+    Given user is at Topology page
+    And user has a workload of Knative Service type resource
+    When user right clicks on the Service to open the Context Menu
+    And user clicks on Edit Health Checks
+    And user sees Readiness Probe already added
+    And user clicks on Add Liveness Probe
+    And user clicks on right button
+    And user clicks on Add button
+    Then user will be redirected to Topology page
+    And user will see Startup Probe added on the Add Health Checks page
+
+
+@smoke, @regression
+Scenario: Edit Health Checks option from Actions dropdown on Sidebar for Deployment Configs to delete a Health Check: MH-05-TC02
+    Given user is at Topology page
+    And user has a workload of Deployment Config type resource with add Health Checks added
```

Comment:
Mention which health check has been added like you are removing the readiness probe so mention it over here

Reply:
The statement was actually "all Health Checks added".

Reply:
Updated